docs/models/virtualization/vminterface.md

```diff
@@ -1,6 +1,6 @@
 ## Interfaces
 
-[Virtual machine](./virtualmachine.md) interfaces behave similarly to device [interfaces](../dcim/interface.md): They can be assigned to VRFs, may have IP addresses, VLANs, and services attached to them, and so on. However, given their virtual nature, they lack properties pertaining to physical attributes. For example, VM interfaces do not have a physical type and cannot have cables attached to them.
```

Comment:
I removed this bit here because I could not find any direct relationship between application services and VM interfaces. If this is some other concept of service, let me know and I can add it back.